pkgs/applications/graphics/oculante/default.nix

```diff
@@ -44,19 +46,23 @@ rustPlatform.buildRustPackage rec {
   buildInputs = [
     openssl
     fontconfig
-  ] ++ lib.optionals stdenv.isLinux [
+   ] ++ lib.optionals stdenv.isLinux [
+    libGL
+    libX11
     libX11
     libXcursor
     libXi
     libXrandr
-    libGL
     gtk3
+
+    libxkbcommon
+    wayland
   ] ++ lib.optionals stdenv.isDarwin [
     darwin.libobjc
   ];
 
   postFixup = lib.optionalString stdenv.isLinux ''
-    patchelf $out/bin/oculante --add-rpath ${lib.makeLibraryPath [ libGL ]}
+    patchelf $out/bin/oculante --add-rpath ${lib.makeLibraryPath ([] ++ lib.optionals stdenv.isLinux [ libxkbcommon libX11 ])}
```

Comment:
```suggestion
    patchelf $out/bin/oculante --add-rpath ${lib.makeLibraryPath [ libxkbcommon libX11 ]}
```

Reply:
why remove that isLinux check though?

Reply:
`postFixup` already only applies to `stdenv.isLinux`

Reply:
oh, TIL, thanks, will push in a minute.

Reply:
it is only the case because of

https://github.com/NixOS/nixpkgs/blob/1a411f23ba299db155a5b45d5e145b85a7aafc42/pkgs/applications/graphics/oculante/default.nix#L56

this is there because `patchelf` doesn't work on darwin